.travis.yml

```diff
@@ -24,6 +24,28 @@ matrix:
       env: TOXENV=pep8py2
     - python: 3.6
       env: TOXENV=pep8py3
+    - python: 3.6
+      env: TOXENV=py36
+      apt:
+        packages:
+          - git
+          - gcc
+          - libssl-dev
+          - gnutls-bin
+      before_script:
+        - wget https://dist.opendnssec.org/source/softhsm-2.5.0.tar.gz \
```

Comment:
Please add a script that check if softhsm is already installed and skips installing it

Reply:
Surrounded this with a `type -p` check...